core/Cargo.toml

```diff
@@ -295,14 +295,16 @@ sha2 = { version = "0.10", optional = true }
 sled = { version = "0.34.7", optional = true }
 suppaftp = { version = "5.2", default-features = false, features = [
   "async-secure",
+  "rustls",
   "async-rustls",
-  "async-native-tls",
 ], optional = true }
 tikv-client = { version = "0.3.0", optional = true, default-features = false }
 tokio = "1.27"
 tokio-postgres = { version = "0.7.8", optional = true }
 tracing = { version = "0.1", optional = true }
 uuid = { version = "1", features = ["serde", "v4"] }
+webpki-roots = { version = "0.22.4", optional = true }
```

Comment:
Can we remove them? Since we don't use them in our code directly.

Reply:
I have removed this.